src/components/test-utils/mocks/MockBrowserRouter.tsx

```
@@ -0,0 +1,13 @@
+import React, { ReactNode } from 'react';
+
+/**
+ * A mock implementation of BrowserRouter for testing purposes.
+ * Wraps children in a div with a test ID for verification in tests.
+ *
+ * @param props - Component properties
+ * @param props.children - Child elements to be wrapped by the router
```

ESLint

[error] tsdoc-param-tag-with-invalid-name: The @param block should be followed by a valid parameter name: The identifier cannot non-word characters

(tsdoc/syntax)